package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "html-react-parser",
-  "version": "0.4.7",
+  "version": "0.4.8",
```

Comment:
**revert:** undo version bump

Reply:
tip: If just *you* run it, try to not keep it in the repo, If someone try to run all npm scripts to ensure 'all' things have done, comes to this.